nucypher/blockchain/eth/providers.py

```diff
@@ -14,25 +14,33 @@ class ProviderError(Exception):
     pass
 
 
-def _get_IPC_provider(eth_provider_uri) -> BaseProvider:
-    uri_breakdown = urlparse(eth_provider_uri)
+def _get_IPC_provider(blockchain_endpoint) -> BaseProvider:
```

Comment:
Can be simplified to just `endpoint: str` IMO.  There are several more usages below as well.

```suggestion
def _get_IPC_provider(endpoint: str) -> BaseProvider:
```

Reply:
Do you mean just for this module or in general?

Reply:
Some spots like `providers.py`/`interfaces.py` (perhaps others...) it is probably redundant to use the `blockchain_` prefix. In some other areas, I think it does provide clarity.

Made an initial change to `providers.py`/`interfaces.py`.